BuildAndTest.cmd

```diff
@@ -37,48 +37,35 @@ echo         public const string Version = AssemblyVersion + Prerelease;
 echo     }                                                                           >> %VERSION_CONSTANTS%
 echo  }                                                                              >> %VERSION_CONSTANTS%
 
-::Restore packages
+:: Restore packages
 echo Restoring packages...
 dotnet restore %~dp0src\BinSkim.sln /p:Configuration=%Configuration% --configfile "%NuGetConfigFile%" --packages "%NuGetPackageDir%
 
 :: Build the solution 
 echo Building solution...
 dotnet build --no-restore /verbosity:minimal %~dp0src\BinSkim.sln /p:Configuration=%Configuration% /filelogger /fileloggerparameters:Verbosity=detailed || goto :ExitFailed
 
-::Run unit tests
-echo Run all multitargeting xunit tests
-call :RunTestProject BinaryParsers Unit || goto :ExitFailed
-call :RunTestProject BinSkim.Rules Unit || goto :ExitFailed
-call :RunTestProject BinSkim.Driver Functional || goto :ExitFailed
-call :RunTestProject BinSkim.Rules Functional  || goto :ExitFailed
+:: Run unit tests
+echo Running tests...
+dotnet test %~dp0src\BinSkim.sln /p:Configuration=%Configuration% --no-build

```

Comment:
If we create more test projects, we would need to update this script every time. With this, we won't need that change